src/gui/qgsmaplayerloadstyledialog.cpp

```diff
@@ -76,6 +76,7 @@ QgsMapLayerLoadStyleDialog::QgsMapLayerLoadStyleDialog( QgsMapLayer *layer, QWid
     updateLoadButtonState();
   } );
   mStyleTypeComboBox->addItem( tr( "From File" ), QgsVectorLayerProperties::QML ); // QML is used as entry, but works for SLD too, see currentStyleType()
+  mStyleTypeComboBox->addItem( tr( "Default from local database" ), QgsVectorLayerProperties::LOCAL );
```

Comment:
```suggestion
  mStyleTypeComboBox->addItem( tr( "Default from Local Database" ), QgsVectorLayerProperties::LOCAL );
```

Combobox entries should be Title Case

Reply:
```suggestion
  mStyleTypeComboBox->addItem( tr( "Default From Local Database" ), QgsVectorLayerProperties::LOCAL );
```